pkg/cli/admin/release/package.go

```diff
@@ -0,0 +1,156 @@
+			referenced via its content digest, which allows this command and the update infrastructure to
+			validate that updates have not been tampered with.
+
+			If no arguments are specified the release of the currently connected cluster is displayed.
+			Specify one image to see details of each release image. You may also pass a semantic version
+			(4.11.2) as an argument, and if cluster version object has seen such a version in the upgrades
+			channel it will find the release info for that version.
+
+			If the specified image supports multiple operating systems, the image that matches the
+			current operating system will be chosen. Otherwise you must pass --filter-by-os to
+			select the desired image.
+		`),
+		Example: templates.Examples(`
+			# Show information about the cluster's current release
+			oc adm release package
+
+			# Show the information about a specific release
+			oc adm release package 4.11.2
+
+			# Show information about linux/s390x image
+			# Note: Wildcard filter is not supported. Pass a single os/arch to extract
+			oc adm release info quay.io/openshift-release-dev/ocp-release:4.11.2 --filter-by-os=linux/s390x
+
+		`),
+		Run: func(cmd *cobra.Command, args []string) {
+			kcmdutil.CheckErr(o.Complete(f, cmd, args))
+			kcmdutil.CheckErr(o.Validate())
+			kcmdutil.CheckErr(o.Run())
+		},
+	}
+	flags := cmd.Flags()
+	o.SecurityOptions.Bind(flags)
+	o.FilterOptions.Bind(flags)
+	o.ParallelOptions.Bind(flags)
+	o.KubeTemplatePrintFlags.AddFlags(cmd)
+
+	flags.StringVar(&o.ICSPFile, "icsp-file", o.ICSPFile, "Path to an ImageContentSourcePolicy file. If set, data from this file will be used to find alternative locations for images.")
+	return cmd
+}
+
+type PackageOptions struct {
+	genericclioptions.IOStreams
+	genericclioptions.KubeTemplatePrintFlags
+
+	Images []string
+	Image  string
+	From   string
+
+	Output   string
+	ImageFor string
+	Verify   bool
+	ICSPFile string
+
+	ParallelOptions imagemanifest.ParallelOptions
+	SecurityOptions imagemanifest.SecurityOptions
+	FilterOptions   imagemanifest.FilterOptions
+}
+
+func (o *PackageOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args []string) error {
+	args, err := findArgumentsFromCluster(f, args)
+	if err != nil {
+		fmt.Println(strings.Replace(err.Error(), "info", "package", 1))
+	}
+	if len(args) < 1 {
+		return fmt.Errorf("packge expects at least one argument, a release image pull spec")
+	}
+	o.Images = args
+	if len(o.From) == 0 && len(o.Images) == 2 && !o.Verify {
+		o.From = o.Images[0]
+		o.Images = o.Images[1:]
+	}
+	return o.FilterOptions.Complete(cmd.Flags())
+}
+
+func (o *PackageOptions) Validate() error {
+	return o.FilterOptions.Validate()
+}
+
+func (o *PackageOptions) Run() error {
+	var exitErr error
+	image := strings.Join(o.Images, " ")
+	infoopts := NewInfoOptions(o.IOStreams)
+	infoopts.SecurityOptions = o.SecurityOptions
+	infoopts.FilterOptions = o.FilterOptions
+	infoopts.ImageFor = "machine-os-content"
```

Comment:
machine-os-content is deprecated and will anytime go away from release image, see https://issues.redhat.com/browse/MCO-392 . Instead, you should use rhel-coreos (contains base RHCOS content) and rhel-coreos-extensions (contains OCP supported extensions and RT kernel packages) which is available from 4.13.

Reply:
in 4.12, they are referenced as rhel-coreos-8 and rhel-coreos-8-extensions. So, based on which all releases this impact, changes can be handled accordingly.

Reply:
Thanks @sinnykumari, for the suggestion. I will add that changes.